src/middlewared/middlewared/plugins/smb.py

```diff
@@ -1325,7 +1325,7 @@ async def validate(self, data, schema_name, verrors, old=None):
                     'This feature may be enabled in the general SMB server configuration.'
                 )
 
-    @api_method(SmbSharePrecheckArgs, SmbSharePrecheckResult, roles=['READONLY_ADMIN'], private=True)
+    @api_method(SmbSharePrecheckArgs, SmbSharePrecheckResult, private=True)
```

Comment:
Let's not remove roles before verifying that it won't break downstream consumers.

Reply:
Which consumers? It's not a part of the API.